src/runtime_src/core/common/query_requests.h

```diff
@@ -698,6 +699,12 @@ struct xclbin_uuid : request
 
   virtual boost::any
   get(const device*) const = 0;
+  
+  static bool
+  is_empty(const std::string& uuid) 
+  {
+    return (boost::equals(uuid, "00000000-0000-0000-0000-000000000000") || boost::equals(uuid, ""));
+  }
```

Comment:
Remove

Reply:
And here Shivangi and I though we invented the wheel! Cool feature with the boolean overload on the uuid :)